src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -0,0 +1,1507 @@
+            string separator = NumberFormatInfo.GetInstance(formatProvider).NumberGroupSeparator;
+            sb.Append('<');
+            for (int index = 0; index < Count - 1; index++)
+            {
+                sb.Append(((IFormattable)GetElement(index)).ToString(format, formatProvider));
+                sb.Append(separator);
+                sb.Append(' ');
+            }
+            // Append last element w/out separator
+            sb.Append(((IFormattable)GetElement(Count - 1)).ToString(format, formatProvider));
+            sb.Append('>');
+            return sb.ToString();
+        }
+
+        /// <summary>
+        /// Attempts to copy the vector to the given <see cref="Span{Byte}" />.
+        /// The destination span must be at least size <see cref="Vector{Byte}.Count" />.
+        /// </summary>
+        /// <param name="destination">The destination span which the values are copied into</param>
+        /// <returns>True if the source vector was successfully copied to <paramref name="destination" />. False if
+        /// <paramref name="destination" /> is not large enough to hold the source vector.</returns>
+        public readonly bool TryCopyTo(Span<byte> destination)
+        {
+            ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();
+
+            if ((uint)destination.Length < (uint)Vector<byte>.Count)
+            {
+                return false;
+            }
+
+            Unsafe.WriteUnaligned<Vector<T>>(ref MemoryMarshal.GetReference(destination), this);
+            return true;
+        }
+
+        /// <summary>
+        /// Attempts to copy the vector to the given <see cref="Span{T}" />.
+        /// The destination span must be at least size <see cref="Count" />.
+        /// </summary>
+        /// <param name="destination">The destination span which the values are copied into</param>
+        /// <returns>True if the source vector was successfully copied to <paramref name="destination" />. False if
+        /// <paramref name="destination" /> is not large enough to hold the source vector.</returns>
+        public readonly bool TryCopyTo(Span<T> destination)
+        {
+            ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();
+
+            if ((uint)destination.Length < (uint)Count)
+            {
+                return false;
+            }
+
+            Unsafe.WriteUnaligned<Vector<T>>(ref Unsafe.As<T, byte>(ref MemoryMarshal.GetReference(destination)), this);
+            return true;
+        }
+
+        /// <summary>Adds two vectors together.</summary>
+        /// <param name="left">The first source vector.</param>
+        /// <param name="right">The second source vector.</param>
+        /// <returns>The summed vector.</returns>
+        [Intrinsic]
+        public static unsafe Vector<T> operator +(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
```

Comment:
As with the `IsHardwareAccelerated` path in the previous code, this just does a `for` loop from `index to Count` and calls the relevant `Scalar*` operation.
It just uses `SetElement` and `GetElement` now, rather than using stackalloc and the indexer.